index.js

```diff
@@ -5,17 +5,15 @@ var _ = require("lodash");
 
 var envName = process.env.NODE_ENV || "development";
 var basePath = process.env.CONFIG_BASE_PATH || process.cwd();
+var configDir = process.env.CONFIG_DIR || "config";
```

Comment:
actually it was the simplest way to test. So it's not an intended feature, I guess the same goes for CONFIG_BASE_PATH?
Should I rewrite the tests so that they work without that addition?